engine/interpreter/src/main/scala/pl/touk/nussknacker/engine/TypeDefinitionSet.scala

```diff
@@ -0,0 +1,44 @@
+package pl.touk.nussknacker.engine
+
+import org.springframework.expression.spel.SpelNode
+import pl.touk.nussknacker.engine.definition.{DefinitionExtractor, ProcessDefinitionExtractor, TypeInfos}
+
+
+case class TypeDefinitionSet(typeDefinitions: Set[TypeInfos.ClazzDefinition] = Set()) {
+
+  def validateTypeReference(spelNode: SpelNode) : Boolean = {
+
+
+    if(spelNode.getChildCount < 1) {
+      //todo: przechwycić błąd w razie jakby nie miało jednego child node'a
+    }
+
+    val spelNodeChildAST = spelNode.getChild(0).toStringAST
```

Comment:
You can do something like this: `spelNode.children..headOption.getOrElse(throw Exception(""))`

Reply:
changed it, thanks